metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -562,35 +591,19 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
 
         discovered_datasets = discovered_tables + discovered_views
 
-        if self.config.include_table_lineage:
+        if self.config.include_table_lineage and self.lineage_extractor:
+            self.report.set_ingestion_stage("*", LINEAGE_EXTRACTION)
             yield from self.lineage_extractor.get_workunits(
                 discovered_tables=discovered_tables,
                 discovered_views=discovered_views,
                 schema_resolver=self.sql_parser_schema_resolver,
                 view_definitions=self.view_definitions,
             )
+            # TODO -  Update the checkpoint state for this lineage run if usage extraction is successful.
```

Comment:
I think this comment can be removed, as this is done in the lineage extractor. Unless you're actually talking about usage here?

Reply:
yeah.